games-fps/eduke32/eduke32-20180224_6694.recipe

```diff
@@ -31,13 +31,13 @@ years ago
 * EDuke32 makes sandwiches!"
 HOMEPAGE="http://www.eduke32.com/"
 COPYRIGHT="1993-1997 Ken Silverman
-	2006-2017 eduke32 Team"
+	2006-2018 eduke32 Team"
 LICENSE="GNU GPL v2"
```

Comment:
From [`HOMEPAGE`][0]:

> EDuke32 is licensed under the [GNU GPL][1] and the [BUILD][2] license.

```diff
-LICENSE="GNU GPL v2"
+LICENSE="GNU GPL v2
+	BUILD"
```

[0]: http://www.eduke32.com
[1]: http://www.gnu.org/licenses/gpl2.txt
[2]: http://eduke32.com/buildlic.txt

Reply:
Hmm, we can't call this license "BUILD". It needs a more specific name, e.g. `Advsys.net Ken Silverman`.
And we need to include that file in `games-fps/eduke32/licenses/`.

Reply:
OK, I just noticed the license is there.
But it needs to be renamed because it would not be acceptable to have that license in `/system/data/licenses/BUILD`.

Hmm, instead of renaming that license, we could move it to `$docDir` (and remove it from the `LICENSE` variable). That would keep `/system/data/licenses/` clean.